aws_lambda_powertools/utilities/batch/sqs_fifo_partial_processor.py

```diff
@@ -1,8 +1,11 @@
-from typing import List, Optional, Tuple
+import logging
+from typing import Optional, Set
 
-from aws_lambda_powertools.utilities.batch import BatchProcessor, EventType
+from aws_lambda_powertools.utilities.batch import BatchProcessor, EventType, ExceptionInfo, FailureResponse
 from aws_lambda_powertools.utilities.batch.types import BatchSqsTypeModel
 
+logger = logging.getLogger(__name__)
+
 
 class SQSFifoCircuitBreakerError(Exception):
```

Comment:
What do you think about moving this Exception to `exceptions.py` file?

Reply:
Don't you think that might be a breaking change if we mess up with the import path?

Reply:
I don't think! If you're importing from `exception.py` in the `sqs_fifo_partial_processor.py`, customers can still import it in the same manner.